contracts/CollectionParty.sol

```diff
@@ -166,6 +183,9 @@ contract PartyBuy is Party {
     */
     function getMaximumContributions() public view returns (uint256 _maxContributions) {
         uint256 _price = maxPrice;
+        if (_price == 0) {
```

Comment:
do we need this? is this just for the front-end?

Reply:
setting price to zero is what removes the maxPrice limit. if maxPrice is set to zero, we return MAX_INT